services/brew-registry-pullsecret/admin_rbac.yaml

```diff
@@ -27,3 +27,36 @@ subjects:
 - kind: Group
   apiGroup: rbac.authorization.k8s.io
   name: brew-registry-pullsecret-admins
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
I don't think this needs a `ClusterRole`, AFAIK a `Role` in your namespace should be enough.